lib/nx_zlib.h

```diff
@@ -93,6 +93,10 @@
 void nx_print_dde(nx_dde_t *ddep, const char *msg);
 #endif
 
+#define zlib_version zlibVersion()
+
+ZEXTERN const char * ZEXPORT zlibVersion OF((void));
```

Comment:
We don't use `ZEXPORT` and we use `extern` instead of `ZEXTERN`.

Reply:
Fixed!